src/main/java/blobExample.txt

```diff
@@ -0,0 +1 @@
+Hello World!
```

Comment:
Can you add the target directory to the gitignore as well and remove the files in target in your PR?